pennylane/operation.py

```diff
@@ -547,24 +547,18 @@ def hyperparameters(self):
     def decomposition(*params, wires):
```

Comment:
It's getting late, but should this not be an instance method that only takes `self`? In fact, I wonder how you can call `self` below? (Is maybe my github view corrupted?)

Reply:
I think that was a problem with github merging things.  I've cleaned that up locally